myhpom/templates/myhpom/upload/current_ad.html

```diff
@@ -4,11 +4,53 @@
 <li class="advance-directive-widget__breadcrumb-status">
     current status:
 </li>
-<li class="advance-directive-widget__breadcrumb-status-value">
-    Saved
-</li>
+{% with ad=request.user.advancedirective %}
+{% if ad.verification_passed %}
+<li class="advance-directive-widget__breadcrumb-status-value--success">Verified</li>
+{% elif ad.verification_in_progress %}
+<li class="advance-directive-widget__breadcrumb-status-value--inprogress">Verifying</li>
+{% elif ad.verification_result %}
+<li class="advance-directive-widget__breadcrumb-status-value--error">Invalid</li>
+{% elif ad.verification_failed %}
+{% comment %}
+a slight variation on language so that support people can differentiate between
+the status of this failure -- which in all likelyhood will require a developer
+resubmitting the AD for verification.
+{% endcomment %}
+<li class="advance-directive-widget__breadcrumb-status-value--error">In verification</li>
+{% else %}
+<li class="advance-directive-widget__breadcrumb-status-value">Saved</li>
+{% endif %}
+{% endwith %}
 {% endblock widget_breadcrumb_items %}
 
+{% block widget_breadcrumb_detail %}
+{% with ad=request.user.advancedirective %}
+{% if ad.verification_passed %}
+{% elif ad.verification_result %}
+    {% if ad.verification_result.owner_name_matches not in yes_or_na %}
+        Name on document doesn't match yours.
+    {% elif ad.verification_result.witness_signature_1 not in yes_or_na %}
+        Missing first witness signature.
+    {% elif ad.verification_result.witness_signature_2 not in yes_or_na %}
+        Missing second witness signature.
+    {% elif ad.verification_result.notarized not in yes_or_na %}
+        Not notarized.
+    {% elif ad.verification_result.signed_by_owner not in yes_or_na %}
+        Not signed.
+    {% else %}
+        Could not be verified. Please re-scan and upload.
+    {% endif %}
+{% elif ad.verification_failed %}
+    Please re-scan and upload.
```

Comment:
Did we agree (you, me, gannon on slack) that an error (probably ours) during the run process should not prompt the user to re-upload their document?

Reply:
No no agreement about any of this language. The ticket is pretty vague - I'll post what this looks like on the PR now and get Gannon's input.